stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-14.0
+resolver: lts-18.13
```

Comment:
I figure it makes sense to develop off of the latest (the latest `lts` that is, not nightly).